src/Gui.java

```diff
@@ -15,18 +13,20 @@ public class Gui implements ActionListener {
     JMenuItem verzoekItem = new JMenuItem("Verzoek");
     JMenuItem authorisatieItem = new JMenuItem("Authorisatie");
     JMenuItem instellingenItem = new JMenuItem("Instellingen");
+    JMenuItem closeItem = new JMenuItem("x");
+    JMenuItem minimizeItem = new JMenuItem("-");
 
     public Gui(){
-        KadasterGui = new JFrame("KadasterGui");
-        KadasterGui.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
moet deze er niet nog steeds in?

Reply:
nee die hoeft er niet in aangezien ik het scherm nu via zichzelf controleer want dat ziet er beter uit